### ISSUE TITLE
santizing "album_title" before saving the Cover Art

### ISSUE THREAD

**Issue by viulian** [blackbunt/dump-discogs-collection-2-csv#7]
Hello,

Have a look at this release please: https://www.discogs.com/release/95971-Boney-M-Ma-Baker-Still-Im-Sad

The album title is "Ma Baker / Still I'm Sad" - but the code won't save the file due to '/' being present in the Album Title.

Thank you,

**Comment by blackbunt** [comment]
hello, thanks for the hint, i am very busy with private stuff, so this can take a while, but i try to fix it!

**Comment by viulian** [comment]
I have created a PR with a proposed fix. 

The code in this repo allows downloading the Cover Art for each of the releases in your collection (if you type 5 at "Choose your Option"). It just didn't show it in the list of options. 

However, when choosing option 5, most Cover Art images (the small ones) are downloaded - except those for which the Album or Artist Name contain invalid file system characters (such as '/' on Linux) - where the files can't be created. The script doesn't crash, except, the file is not present in the `Cover-Art` folder.

Thus, the PR adds the option 5 and make it visible, and it also sanitizes the Artist / Album Name used when saving the cover art, by replacing all non-ASCII characters with '_'. I did not check other possible places in the code where file name "sanitization" may be needed.
